test/TestBase.py

```diff
@@ -104,6 +104,7 @@ class WorldTestBase(unittest.TestCase):
     multiworld: MultiWorld
 
     game: typing.ClassVar[str]  # define game name in subclass, example "Secret of Evermore"
+    player: typing.ClassVar[int] = 1
```

Comment:
A bit odd to have these changes in this PR, they do not share the same topic.

Reply:
It makes some of the tests easier to understand if we use `self.player` instead of hard coding 1. I can move that in another PR if you want